homeassistant/components/shelly/switch.py

```diff
@@ -49,15 +49,7 @@ def is_on(self) -> bool:
     @property
     def device_info(self):
         """Device info."""
-        if not self.multiple_blocks:
-            return super().device_info
-
-        # If a device has multiple relays, we want to expose as separate device
-        return {
-            "name": self.name,
-            "identifiers": {(DOMAIN, self.wrapper.mac, self.block.index)},
-            "via_device": (DOMAIN, self.wrapper.mac),
-        }
+        return super().device_info
```

Comment:
No need to override it anymore, just remove it completely. 

Also, remove the `self.multiple_blocks` as it's no longer used.

Reply:
Code cleaned